include/parakeet/Driver.h

```diff
@@ -112,80 +148,39 @@ class Driver
             unsigned char intensity[1000];
```

Comment:
I'm assuming 1000 is the maximum number of points that can be received? Can we make that a const instead of using the raw number twice?

Reply:
That is the same assumption I have. I'm fine with making it a const.